docs/deployment/deployment.md

```diff
@@ -0,0 +1,67 @@
+`modelkit` centralizes all of your models in a single object, which makes it easy to serve as a REST API via an HTTP server.
```

Comment:
```suggestion
`modelkit` centralizes all of your models in a single object, which makes it easy to serve as a REST API via a HTTP server.
```

Reply:
Nit: "an HTTP" seems to be used all over the place, so let's clean this out in another PR

Reply:
It is correct to my ears, because HTTP starts with a vowel phoneme like "eight" rather than an aspirated h like "hate". But I'm not 100% sure.